MapboxCoreNavigation/NavigationEventsManager.swift

```diff
@@ -85,9 +85,8 @@ open class NavigationEventsManager: NSObject {
     func start() {
         let eventLoggingEnabled = UserDefaults.standard.bool(forKey: NavigationMetricsDebugLoggingEnabled)
         
-        mobileEventsManager.isDebugLoggingEnabled = eventLoggingEnabled
-        mobileEventsManager.isMetricsEnabledInSimulator = true
-        mobileEventsManager.isMetricsEnabledForInUsePermissions = true
+        
+        UserDefaults.standard.mme_isCollectionEnabled = eventLoggingEnabled
```

Comment:
This still needs a change. Either replace `UserDefaults.standard.mme_isCollectionEnabled` with some implementation of `mobileEventsManager.isDebugLoggingEnabled` or remove entirely for the time being. `mme_isCollectionEnabled` should be handled by the Events Library. 

We'll add a plist flag for debug logging in a later release.

Reply:
cc: @alfwatt

Reply:
I think we should remove this bit, as well as the `NavigationMetricsDebugLoggingEnabled` constant. This debug logging is an advanced workflow that is only relevant when we develop more event types, so we could just enable it locally when needed without involving the SDK.